hawc/apps/eco/lookups.py

```diff
@@ -0,0 +1,75 @@
+from selectable.base import ModelLookup
+from selectable.registry import registry
+
+from ..common.lookups import RelatedDistinctStringLookup
+from ..epi.models import Country
+from . import constants, models
+
+
+class RelatedCauseLookup(RelatedDistinctStringLookup):
```

Comment:
update these to the new [`dal` ](https://github.com/shapiromatron/hawc/pull/667) autocompletes